src/plaquette/codes/__init__.py

```diff
@@ -112,11 +116,8 @@ def stabilizers(self) -> t.Sequence[Tableau]:
         """
         checks: list[Tableau] = list()
         for op_idxs in self.factorized_checks:
-            checks.append(
-                pauli.multiply(
-                    np.vstack([self.measured_operators[o] for o in op_idxs])
-                )[0]
-            )
+            stacked = [self.measured_operators[o] for o in op_idxs]
+            checks.append(pauli.multiply(*stacked)[0])
```

Comment:
Minor suggestion: the name is now outdated
```suggestion
            list_of_ops = [self.measured_operators[o] for o in op_idxs]
            checks.append(pauli.multiply(*list_of_ops)[0])
```